test/scripts/openshift-ci/deploy.odh.sh

```
@@ -12,110 +12,103 @@
 
-# Create PVC for custom manifests
-echo "Creating PVC for custom KServe manifests..."
-oc apply -f "${SCRIPT_DIR}/odh-operator-custom-manifests/pvc.yaml"
+  echo "Patching operator CSV to mount custom manifests volume..."
+  CSV=$(oc get subscription "${OPERATOR_NAME}" -n "${OPERATOR_NAMESPACE}" -o jsonpath='{.status.installedCSV}')
+  echo "Found CSV: $CSV"
 
-# Note: PVC may stay in Pending state with WaitForFirstConsumer binding mode
-# It will only bind when a pod (the operator) actually consumes it
-echo "PVC created (will bind when consumed by operator pod)"
+  if oc get csv "$CSV" -n ${OPERATOR_NAMESPACE} -o json | jq -e '.spec.install.spec.deployments[0].spec.template.spec.volumes[] | select(.name=="kserve-custom-manifests")' > /dev/null 2>&1; then
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)